about.html

```diff
@@ -44,6 +44,7 @@
             </a>
```

Comment:
- [ ] The hamburger menu button doesn't work on the about page. When I click on the hamburger icon, the menu does not show up. Kindly fix the issue.